scripts/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+figlet PiKaraoke
+
+# Run pikaraoke with necessary parameters
+cd pikaraoke
+poetry run pikaraoke -d /pikaraoke-songs/ --headless --high-quality
```

Comment:
Would prefer not to force the `--high-quality` flag. Not sure if there's a way to passthrough command line parameters using docker, but that would be preferred

Reply:
We can definitely do that, the way you specify features like that is usually via environment variables (the `env:` block that is currently not included. Question is, do we want to just have one environment variable that contains all the extra params, or do you want one for each option?

so we'd either have

```
env:
    LAUNCH_OPTIONS: '--high-quality'
```

or

```
env:
   HIGH_QUALITY: true
```